tests/reproducibility/fixtures.py

```diff
@@ -23,13 +23,13 @@ def models_equals(model1, model2):
     return all(np.array_equal(w, w_new) for w, w_new in zip(w1, w2))
 
 
-def assert_reproducible(func, num_iter=20):
+def assert_reproducible(func, num_iter=10):
```

Comment:
I've had to reduce this since the tests take too long to run. On my local machine, with `num_iter=20`, the slowest reproducibility tests:
```
====================================== slowest 4 test durations =======================================
25.36s call     tests/reproducibility/test_graphsage.py::test_unsupervised[True]
24.63s call     tests/reproducibility/test_graphsage.py::test_unsupervised[False]
20.32s call     tests/reproducibility/test_graphsage.py::test_link_prediction[False]
18.54s call     tests/reproducibility/test_graphsage.py::test_link_prediction[True]
```

and with `num_iter=10`:
```
====================================== slowest 4 test durations =======================================
14.47s call     tests/reproducibility/test_graphsage.py::test_unsupervised[True]
13.79s call     tests/reproducibility/test_graphsage.py::test_unsupervised[False]
9.69s call     tests/reproducibility/test_graphsage.py::test_link_prediction[False]
9.18s call     tests/reproducibility/test_graphsage.py::test_nai[False]
```
I think 10 should still be enough? There's a lot of setup time in building a tensorflow model that we can't avoid, so we might have to consider some other options later.

Reply:
I guess we could reduce the sizes of the models (e.g. `layer_sizes`) if initialisation is proportional to that?

Reply:
It's a nice idea... but I just tried it and unfortunately it doesn't seem to make any difference 🤔